packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -560,8 +577,8 @@ class _AndroidViewState extends State<AndroidView> {
   }
 }
 
-class _UiKitViewState extends State<UiKitView> {
-  UiKitViewController? _controller;
+abstract class _DarwinViewState<TPlatformView extends _DarwinView, TController extends DarwinPlatformViewController, TRender extends RenderDarwinPlatformView<TController>, TPrivateView extends _DarwinPlatformView<TController, TRender>> extends State<TPlatformView> {
```

Comment:
this line should probably be reformatted into multi lines to make it more readable...

Reply:
yeah `TController` is odd to me. Either use short form or long form in generic, but I've never seen a mix of the 2